### PR TITLE
fix skull facepaint, masks

### PR DESCRIPTION
**Pull request by irakligabunia**
# About the pull request

it fixes the previous PR of adding the skull mask, facepaint

# Explain why it's good for the game

its a fix and sorry, i will test my PR's in the future

# Testing Photographs and Procedure
https://files.catbox.moe/6tyox1.png 

</details>

# Changelog
:cl: stalkerino
fix: fixes the skull facepaint, black mask, blue mask
/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]
unbelivlablele... who would have merged this??

File: code/modules/client/preferences_gear.dm [hunk 1]
```diff
@@ -888,7 +888,7 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
 	display_name = "Facepaint, black"
 	path = /obj/item/facepaint/black
 
-/datum/gear/skullfacepaint
+/datum/gear/misc/facepaint_skull
 	display_name = "Skull Facepaint"
```
Comment (fira) [inline code/modules/client/preferences_gear.dm:892]:
```suggestion
	display_name = "Facepaint, skull"
```

File: code/modules/client/preferences_gear.dm [hunk 2]
```diff
@@ -161,13 +161,13 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
...
-/datum/gear/skull_balaclava_blue
+/datum/gear/mask/uscm/skull_balaclava_blue
 	display_name = "Blue Skull Balaclava"
 	path = /obj/item/clothing/mask/rebreather/skull
 	cost = 4 //same as skull facepaint
 	slot = WEAR_FACE
 
-/datum/gear/skull_balaclava_black
+/datum/gear/mask/uscm/skull_balaclava_black
 	display_name = "Black Skull Balaclava"
```
Comment (fira) [inline code/modules/client/preferences_gear.dm:171]:
```suggestion
	display_name = "USCM balaclava, black skull"
```

File: code/modules/client/preferences_gear.dm [hunk 3]
```diff
@@ -161,13 +161,13 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
 	display_name = "USCM balaclava, tan"
 	path = /obj/item/clothing/mask/rebreather/scarf/tan
 
-/datum/gear/skull_balaclava_blue
+/datum/gear/mask/uscm/skull_balaclava_blue
 	display_name = "Blue Skull Balaclava"
```
Comment (fira) [inline code/modules/client/preferences_gear.dm:165]:
```suggestion
	display_name = "USCM balaclava, blue skull"
```

**Review by fira — CHANGES_REQUESTED** [review]
please standardize names with the rest of gear while you're there

**Review by fira — APPROVED** [review]

**Pull request merged**